documentation/docs/services/SSI-bridge/introduction.md

```diff
@@ -0,0 +1,54 @@
+---
+image: /img/integration-services/arch/network-of-trust.jpeg
+description: The IOTA Ecommerce-SSI Bridge provides an abstraction layer through REST APIs that allows entities to create decentralized identities (DIDs), verify them, attach verifiable credentials and verify them.
+keywords:
+- ecommerce
+- ssi bridge
+- abstraction layer
+- decentralized identity
+- verifiable credentials
+---
+# Ecommerce-SSI Bridge
+
+## Concept and Logic Architecture
+
+The Ecommerce-SSI Bridge allows you to create [Self-Sovereign Identities](../../basics/identity.md), linking
+Decentralized Identifiers (DIDs) to their specification ([DID Documents](https://www.w3.org/TR/did-core/)).
```

Comment:
I'd still decapitalize self-sovereign identities and decentralized identifiers. Btw, they can't make up their mind on w3.org either and capitalize only the first occurence. The glossary entry is decapitalized (possibly something to do with their links formatting conventions)